libs/components/lists/testing/src/modules/paging/page-control-harness.ts

```diff
@@ -0,0 +1,51 @@
+import { ComponentHarness, TestElement } from '@angular/cdk/testing';
+
+/**
+ * Harness to interact with a page control element in tests.
+ */
```

Comment:
Since consumers dont set the page number text and only know it as the 'page number' and not 'page control text', i think we should mark this as internal and use the `SkyPagingHarness` to have functions to just get `getActivePageNumber` and `clickPageNumber` etc

Reply:
I don't like this approach because it's not immediately clear how they should interact with the controls if they're obfuscated behind the paging harness. Do they ask to click a specific page number, which may not be in the list? Do they ask to click the Nth page control, and if so how do we enforce there are only that many controls to begin with? Giving them an array of the paging controls that exist feels very intuitive for them exploring the structure of the page to me. They get, say, 5 test harness elements, they can do a sanity check that they are paging in the way they expect, and interact with them the same way a user would.